datamanagement/miscellaneous/py2hdfread/h5_to_csv.py

```diff
@@ -0,0 +1,225 @@
+
+import time
+import os
+import logging
+import pandas as pd
+import numpy as np
+#import yaml
+import ruamel.yaml as yaml
+from pandas.testing import assert_frame_equal
+from ruamel.yaml.scanner import ScannerError
+import logging
+
+
+
+logging.basicConfig(
+   format='%(levelname)s:%(asctime)s:%(message)s',
+   datefmt='%y%m%d-%H%M%S',
+   level=logging.INFO)
+logger = logging.getLogger('')
+#logger.setLevel(logging.INFO)
+
+root = '/Users/ogolovko/w/bccrc/tt'
+
+pandas2std_types = {
+     'bool'   : 'boolean'
+    ,'int64'  : 'int'
+    ,'float64': 'float'
+    ,'object' : 'str'}
+
+std2pandas_types = {
+     'boolean' : 'bool'
+    ,'int'     : 'int64'
+    ,'float'   : 'float64'
+    ,'str'     : 'object'}
+
+yaml_pref = '''
+__PIPELINE_INFO__:
+  data_type: generic
+__HEADER__:
+  caller: single_cell_hmmcopy_bin
+  sample_id: SAMPLE_ID_HERE
+  file_format: csv
+'''
+
+gz = '.gz'
+csv = '.csv'
+suff_old = '.h5'
+suff_new = '%s%s' % (csv,gz)
+suff_yaml = '.yaml'
+
+class Timeit:
```

Comment:
remove debug code... if you want to have reporting of timing put that in the fixup scripts that call this code.